Tests/eppo/ClientTests.swift

```diff
@@ -112,13 +138,23 @@ struct AssignmentTestCase : Decodable {
         if self.subjectsWithAttributes != nil {
             return try self.subjectsWithAttributes!.map({
                 try client.getStringAssignment(
-                    $0.subjectKey, self.experiment, $0.subjectAttributes
+                    flagKey: self.experiment,
+                    subjectKey: $0.subjectKey,
+                    subjectAttributes: $0.subjectAttributes,
+                    defaultValue: ""
                 )
             });
         }
 
         if self.subjects != nil {
```

Comment:
consider using [optional binding](https://developer.apple.com/documentation/swift/optional#Optional-Binding) with `if let` here:

```
if let subjects = self.subjects {
   return subjects.map({
```

Reply:
this is a really nice pattern - swift is a language i wish i used more